Testing/ReadTestCase.m

```diff
@@ -0,0 +1,21 @@
+% This function reads in a Test Case and the time from the State Vector and outputs
+% the Flight Inputs
+
+function [FlightInputs] = ReadTestCase(Time, TestCase)
+  
+  % Default Flight Inputs
+  FlightInputs.Pitch = 0;
+  FlightInputs.Throttle = 0;
+  
+  % Interpolate the TestCase Pitch Table for the pitch input
+  PitchTableX = TestCase.PitchTable(:,1);

```

Comment:
It could be a good idea to signify that X (axis) is actually time and Y (axis) is value.

Reply:
Updated the names of the Table suffixes from X and Y to Time and Value